Service/DataService.php

```diff
@@ -347,10 +349,21 @@ public function getBusinessIds(array $keys): array
                             ]
                         ]
 
-                    ]
+                    ],
+                    'size' => 100,
+                    "scroll" => "1m"
                 ]);
-                foreach ($result['hits']['hits'] as $hits) {
-                    $businessKeys[$contentType->getName() . ':' . $hits['_id']] = $hits['_source'][$contentType->getBusinessIdField()] ?? $hits['_id'];
+
+                while (count($result['hits']['hits'] ?? []) > 0) {
+                    foreach ($result['hits']['hits'] as $hits) {
+                        $dataLink = $contentType->getName() . ':' . $hits['_id'];
+                        $businessKeys[$dataLink] = $hits['_source'][$contentType->getBusinessIdField()] ?? $hits['_id'];
+                        $this->cacheBusinessKey[$dataLink] = $businessKeys[$dataLink];
+                    }
+                    $result = $this->client->scroll([
+                        'scroll_id' => $result['_scroll_id'],
+                        'scroll' =>  "1m",
```

Comment:
use a constant for hard coded variables as "1m", and use single quotes for non interpreted strings.

Reply:
82198ac8b1633e42f7125253f0a0c70cf9a235aa